.github/workflows/osx.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: macOS build and test AWS Encryption SDK for C
+
+on:
+  pull_request:
+    branches:
+      - master
+
+
+jobs:
+  OSX:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [macOS-10.14, macos-latest]
+        ssl_ver: [openssl@1.1]
+
+    steps:
+      - run: brew install ${{ matrix.ssl_ver }}
+
+      - name: Checkout PR
+        uses: actions/checkout@v2
+
+      - name: Checkout Dependencies
+        uses: actions/checkout@v2
+        with:
+          repository: "aws/aws-sdk-cpp"
+          ref: "1.7.231"
+          path: "aws-sdk-cpp"
+
+      - name: Build and install aws-sdk-cpp
+        run: |
+          mkdir -p build-aws-sdk-cpp || true
+          mkdir -p install || true
+          cd build-aws-sdk-cpp
+          cmake -G Xcode -DTARGET_ARCH="APPLE" -DBUILD_SHARED_LIBS=ON -DBUILD_ONLY="kms" -DENABLE_UNITY_BUILD=ON ../aws-sdk-cpp
+          xcodebuild -target ALL_BUILD
+          xcodebuild -target install
+
+      - name: Build C-ESDK
+        run: |
+          mkdir build-aws-encryption-sdk-c || true
+          cd build-aws-encryption-sdk-c
+          cmake -G Xcode -DBUILD_SHARED_LIBS=ON -DOPENSSL_ROOT_DIR="/usr/local/opt/openssl@1.1" ../
```

Comment:
Presumably, this should be referencing the matrix value?
I think the right way to inject this would be via an environment variable.

ex:
```yaml
- name: Build C-ESDK
  env:
    OPENSSL_VERSION: ${{ matrix.openssl_version }}
  run: |
    ...
    cmake -G -DBUILD_SHARED_LIBS=ON -DOPENSSL_ROOT_DIR="/usr/local/opt/${OPENSSL_VERSION}" ../
    ...
```

There's also the `format` function, but that feels messy for a large, multi-line, value like this.

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/contexts-and-expression-syntax-for-github-actions#format

Reply:
yup, good call.